### PR TITLE
DoubtRemovers

### PR DESCRIPTION
**Pull request by charlene1804**
https://backlight.dev/edit/1SuMaEt5uxgp62VC8WVP/doubt-removers/src/index.js?branch=doubtremovers%40xLXrNTxtX6duzExlKQ6SJBQ7oDo1

2021-09-07_16h15_57

I created a React component for the landing to take up less space but there is still some alignement issues I need to solve with Tailwind. To be continued.

### CODE REVIEW

**Comment by georges-gomes** [comment]
Nice.

Now that I see this, I think we need something even simpler.
The images are too much.

Something in the lines of 
https://tailwindui.com/components/marketing/sections/feature-sections
Maybe with just some icons.

And let's have a single paragraph per element. 2 is weird.

Thanks

**Comment by charlene1804** [comment]
Rodger that!
I wholeheartedly agree.
Will do tomorrow.

Le mar. 7 sept. 2021 à 23:22, Georges Gomes ***@***.***> a
écrit :

**Comment by charlene1804** [comment]
I reorganized the component to look more like the sample @georges-gomes provided.
2021-09-08_11h16_24
I like it much better that way.
I have an issue with loading my SVGs tho, going ask for help.

**Comment by georges-gomes** [comment]
Give it more margin do it "breath". @thais-divriots  can give you some recommendation I think.

Just copy paste the SVG in the html.
Otherwise I like it, it makes more sense for what it is 👌

**Comment by charlene1804** [comment]
All done I think.
2021-09-08_12h07_45
2021-09-08_12h24_43

File: promise/src/content.js [hunk 1]
```diff
@@ -107,58 +107,59 @@ export default [
...
-          src: imgLanding('nolockin.gif'),
-        },
-        // action: {
-        //   label: 'Bar',
-        //   url: '#',
-        // },
-      },
-    ],
-  },
+  // THIS  ↓  HAS BEEN MOVED to doubt-removers ;)
```
Comment (gqio) [inline promise/src/content.js:110]: could be deleted I think

File: studio.config.json [hunk 2]
```diff
@@ -29,14 +29,35 @@
...
+          "price",
+          "dropdown"
+        ]
+      ],
       [
         "templates",
         [
           "highlight",
           "promise",
+          "doubt-removers",
```
Comment (bashmish) [inline studio.config.json:60]: I love this name!

File: promise/src/content.js (hunk 1 above)
Comment (charlene1804) [reply promise/src/content.js:110]: Thank you I will do that.

File: studio.config.json (hunk 2 above)
Comment (charlene1804) [reply studio.config.json:60]: All credit goes to @georges-gomes 😊

**Review by gqio — APPROVED** [review]

**Pull request merged**